bfapi/service/jobs.py

```diff
@@ -208,11 +208,15 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __genCmd(
-        svc_format: str,
+# The services descended from pzsvc-exec all connect to a CLI (command-line interface) program.
```

Comment:
Two things:

1. The general convention for function documentation is docstrings, ala:

```python
def do_something(x):
    """Take x and multiply it a few times"""
    return x * 3 * 2 * 1
```

2. This comment will go stale eventually as many developers forget to update comments when making code changes.  A descriptive function name should be enough to express what a maintainer needs to know about this function.  I'll defer to industry wisdom on this one:

> “A long descriptive name is better than a short enigmatic name. A long descriptive name is better than a long descriptive comment.” 
> ― Robert C. Martin, Clean Code: A Handbook of Agile Software Craftsmanship